README.md

```diff
@@ -1885,6 +1885,25 @@ NOTE: `callback` is an optional URL for pushover to [ack messages](https://pusho
 | `title`       |   O    | application title (dflt: pushover dflt) |
 | `priority`    |   O    | priority. (dflt: pushover setting)     |
 
+The pushover service will accept either a simple mqtt message and post that to pushover, or a json payload can be passed
+to the service including 1 of 2 image types and an accompanying message.
```

Comment:
What does "1 of 2 image types" mean? Is that a typo?

Reply:
there's two ways you can post an image.  Should I just replace that sentence with "can be passed to the service including a message and an image call"   and leave it up to the examples shown below to show the two difference types?